index.js

```diff
@@ -25,7 +25,7 @@ module.exports = function (filepath, options, callback) {
     }
 
     if (opt['phantomjs-options'] && opt['phantomjs-options'].length) {
-        childArgs.push(opt['phantomjs-options']);
+        childArgs = childArgs.concat(opt['phantomjs-options']);
```

Comment:
This is what is needed but I would suggest using the method that gulp-qunit does.
```js
        if (options['phantomjs-options'] && options['phantomjs-options'].length) {
            if (Array.isArray(options['phantomjs-options'])) {
                childArgs = childArgs.concat(options['phantomjs-options']);
            } else {
                childArgs.push(options['phantomjs-options']);
            }
        }
```

Reply:
You can see the change in gulp-qunit when it was made here as well as the included test.

https://github.com/jonkemp/gulp-qunit/pull/30/files

Please include the test for this as well.